lightning/src/ln/channel.rs

```diff
@@ -4150,6 +4264,21 @@ fn get_v2_channel_reserve_satoshis(channel_value_satoshis: u64, dust_limit_satos
 	cmp::min(channel_value_satoshis, cmp::max(q, dust_limit_satoshis))
 }
 
+#[allow(dead_code)] // TODO(dual_funding): Remove once begin_interactive_funding_tx_construction() is used
+fn add_funding_change_output(
+	change_value: u64, change_script: ScriptBuf,
+	funding_outputs: &mut Vec<OutputOwned>, funding_feerate_sat_per_1000_weight: u32,
+) {
+	let mut change_output = TxOut {
+		value: Amount::from_sat(change_value),
+		script_pubkey: change_script,
+	};
+	let change_output_weight = get_output_weight(&change_output.script_pubkey).to_wu();
+	let change_output_fee = fee_for_weight(funding_feerate_sat_per_1000_weight, change_output_weight);
+	change_output.value = Amount::from_sat(change_value.saturating_sub(change_output_fee));
+	funding_outputs.push(OutputOwned::Single(change_output.clone()));
+}
```

Comment:
Let's just inline this at the call site if it isn't used anywhere else.

Reply:
OK, done